docs/pages/server-access/getting-started.mdx

```diff
@@ -309,7 +376,7 @@ This is a convenient feature that allows for more advanced queries. If an IP add
 `tele-admin` can also execute commands on all Nodes that share a label, vastly simplifying repeated operations. For example, the command:
 
 ```code
-# Run the ls command on all Nodes with a label
+# Run the ls command on all SSH Services with a label
```

Comment:
I know `SSH Service` was suggested to you, but I'm not sure a global replace makes sense.

IMO, 'SSH Service' is the Teleport software that's running to make the "server" accessible to Teleport users.

In this case, the resource that is labeled with `env=example` is more of a generic server resource than a Teleport process.

WDYT @ptgott ?

Reply:
I'd agree with that—the SSH Service is what you run to register and access a server. The server is the resource itself.